test/test.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
+import * as _ from 'lodash';
+import { PinejsClientCore } from '..';
+
+// @ts-expect-error
+const core = new PinejsClientCore();
```

Comment:
Not sure what to do about this, the type defined in `es201*/index.d.ts` is an `abstract class` and you can't create an instance of an abstract class as they are technically incomplete.